packages/geo/src/epsg.ts

```diff
@@ -32,6 +32,7 @@ const EPSGTextMap: Record<string, EpsgCode> = {
 
 export class Epsg {
     static Codes = new Map<number, Epsg>();
+    static TargetCodes = [EpsgCode.Google, EpsgCode.Nztm2000];
```

Comment:
What is a TargetCodes?

are these the only output codes that we are using? Not sure if this should be placed here as its somewhat specific to our implmentation of cogification.

Reply:
Moved to `ProjectionTileMatrixSet`